packages/block-library/src/index.native.js

```diff
@@ -65,6 +65,8 @@ import * as socialLinks from './social-links';
 
 import { transformationCategory } from './transformationCategories';
 
+const ALLOWED_BLOCKS_GRADIENT_SUPPORT = [ 'core/button' ];
```

Comment:
By reading this part, I understand we're starting to adopt the hooks, just for some blocks. Why don't we enable it for all the blocks? How can third-party blocks fit into this?

Reply:
Gradients support is tricky on mobile because we have to add a custom background component to show the gradient and currently only the Button block has it, to avoid showing the gradient option in the background color setting for blocks that can't render it, we'll hide it until we develop a way to enable it for all instead of a case by case basis.